README.md

```diff
@@ -135,6 +135,49 @@ AnyCable, k6, WebSockets, and Yabeda](https://evilmartians.com/chronicles/real-t
 - [k6 Extensions](https://grafana.com/docs/k6/latest/extensions/)
 - [GitHub Topic: xk6](https://github.com/topics/xk6) - Explore k6 extensions tagged with the xk6 label.
 
+### Official
+
+- [xk6-client-tracing](https://github.com/grafana/xk6-client-tracing) - Client for load testing distributed tracing backends.
+- [xk6-dashboard](https://github.com/grafana/xk6-dashboard) - Create a web-based metrics dashboard.
```

Comment:
Given that the web dashboard functionality is already part of core k6 is this extension still needed? Should we deprecate it?

Reply:
This module is integrated into k6 with go vendoring. It can still be used as an extension, on the one hand, it can be developed this way without k6 modifications, and on the other hand, the next version can be tested this way.

I can remove it from this list if it's confusing.

Reply:
I don't see a use case where it should be used as an extension given that it's already a core component. I suggest removing it to avoid confusion.

Reply:
I don't think it would be confusing for anyone, but I removed it.